imap-acct (and creds) CLI crashes with nil pointer when storage modules aren’t started

# Describe the bug

`maddy imap-acct` (and related CLI commands) panic because the CLI loads the config but doesn’t call `Start()` on `LifetimeModule`.

For `storage.imapsql`, this leaves `Back` nil, and `EnableUpdatePipe()` dereferences a nil `UpdateManager`, causing a SIGSEGV.

The server path starts modules correctly, but the CLI does not.

# Steps to reproduce

1. Run maddy in Docker with a config that includes `storage.imapsql local_mailboxes`.
2. Execute:
   docker exec -it maddy-local maddy imap-acct create alice@example.test

# Log files

Panic stack trace:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0xeb5430]

goroutine 1 [running]:
github.com/foxcpp/go-imap-sql.(*Backend).UpdateManager(...)
        github.com/foxcpp/go-imap-sql@v0.5.1-0.20250124140007-8da5567429d5/backend.go:343
github.com/foxcpp/maddy/internal/storage/imapsql.(*Storage).EnableUpdatePipe(...)
        internal/storage/imapsql/imapsql.go:348
github.com/foxcpp/maddy/internal/cli/ctl.openStorage(...)
        internal/cli/ctl/moduleinit.go:96
...
```

# Configuration file

(attach or paste sanitized file)
- Based on Docker config (maddy.conf.docker)
- Uses `storage.imapsql local_mailboxes` with sqlite3 dsn

```
storage.imapsql local_mailboxes {
    driver sqlite3
    dsn imapsql.db
}
```
# Environment information

- maddy v0.9.0 / b1dc3ffed23a473c5c80921b4b1675f3b17dc8c6
- Docker / MacOS / Arm64

--- a/internal/cli/ctl/moduleinit.go
+++ b/internal/cli/ctl/moduleinit.go
@@ -37,6 +37,36 @@ func closeIfNeeded(i interface{}) {
 	}
 }
 
+type managedStorage struct {
+	module.ManageableStorage
+	started bool
+}
+
+func (m *managedStorage) Close() error {
+	if !m.started {
+		return nil
+	}
+	if lm, ok := m.ManageableStorage.(module.LifetimeModule); ok {
+		return lm.Stop()
+	}
+	return nil
+}
+
+type managedUserDB struct {
+	module.PlainUserDB
+	started bool
+}
+
+func (m *managedUserDB) Close() error {
+	if !m.started {
+		return nil
+	}
+	if lm, ok := m.PlainUserDB.(module.LifetimeModule); ok {
+		return lm.Stop()
+	}
+	return nil
+}
+
 func getCfgBlockModule(ctx *cli.Context) (*container.C, module.Module, error) {
 	cfgPath := ctx.String("config")
 	if cfgPath == "" {
@@ -92,6 +122,14 @@ func openStorage(ctx *cli.Context) (module.Storage, error) {
 		return nil, cli.Exit(fmt.Sprintf("Error: configuration block %s is not an IMAP storage", ctx.String("cfg-block")), 2)
 	}
 
+	started := false
+	if lt, ok := storage.(module.LifetimeModule); ok {
+		if err := lt.Start(); err != nil {
+			return nil, err
+		}
+		started = true
+	}
+
 	if updStore, ok := mod.(updatepipe.Backend); ok {
 		if err := updStore.EnableUpdatePipe(updatepipe.ModePush); err != nil && !errors.Is(err, os.ErrNotExist) {
 			fmt.Fprintf(os.Stderr, "Failed to initialize update pipe, do not remove messages from mailboxes open by clients: %v\n", err)
@@ -100,6 +138,11 @@ func openStorage(ctx *cli.Context) (module.Storage, error) {
 		fmt.Fprintf(os.Stderr, "No update pipe support, do not remove messages from mailboxes open by clients\n")
 	}
 
+	if started {
+		if ms, ok := storage.(module.ManageableStorage); ok {
+			return &managedStorage{ManageableStorage: ms, started: started}, nil
+		}
+	}
 	return storage, nil
 }
 
@@ -114,5 +157,16 @@ func openUserDB(ctx *cli.Context) (module.PlainUserDB, error) {
 		return nil, cli.Exit(fmt.Sprintf("Error: configuration block %s is not a local credentials store", ctx.String("cfg-block")), 2)
 	}
 
+	started := false
+	if lt, ok := userDB.(module.LifetimeModule); ok {
+		if err := lt.Start(); err != nil {
+			return nil, err
+		}
+		started = true
+	}
+
+	if started {
+		return &managedUserDB{PlainUserDB: userDB, started: started}, nil
+	}
 	return userDB, nil
 }

